libcaf_core/caf/abstract_actor.cpp

```diff
@@ -189,15 +189,15 @@ void abstract_actor::register_at_system() {
     return;
   setf(is_registered_flag);
   [[maybe_unused]] auto count = home_system().registry().inc_running();
-  CAF_LOG_DEBUG("actor" << id() << "increased running count to" << count);
+  log::core::debug("actor {} increased running count to {}", id(), count);
```

Comment:
I think this line should go to the `system` category (needs `log/system.hpp`), where we log events for spawning / terminating actors.

```suggestion
  log::system::debug("actor {} increased running count to {}", id(), count);
```